### DISCUSSION TITLE
how posts are read

### DISCUSSION THREAD

**Discussion by egy1st** [Vagr9K/gatsby-material-starter#226] [Q&A]
May I ask you, where in this code I can config it to read posts from local markdown files / headless cms

**Comment by Vagr9K** [comment]
This tarter uses [`gatsby-theme-advanced`](https://github.com/Vagr9K/gatsby-advanced-starter/tree/master/themes/advanced) under the hood for managing data.

You can either use the plugin API to configure it or override it's settings in your own `gatsby-config.js`
